snippets/zoe/contracts/test-oracle.js

```diff
@@ -98,11 +99,11 @@ test('oracle contract', async t => {
   t.is(offerResult, 42);
 
   // #region API
-  const oracleHandlerAPI = {
+  const oracleHandlerAPI = Far('oracleHandlerAPI', {
```

Comment:
Why should `oracleHandlerAPI` be marked `Far`?

Reply:
Clarified verbally that it is the kind of thing, like the `oracleHandler` above it, that is normally a remotable, i.e., a passable with remotely callable methods. Admittedly, when I made this change, I didn't realize that only one is used this way. But nevertheless, both are written as if they are intended to be used this way.

Reply:
@kriskowal is this like https://github.com/Agoric/dapp-fungible-faucet/pull/31#issuecomment-903977560 , a RESM to NESM thing?

Reply:
No, I do not believe this is related to RESM/NESM migration.

Reply:
Arg. I added that question to the wrong conversation. Hold on.